string child_frame_id
float64 mass
